data/scrape/lib/generate_summary.py

```diff
@@ -114,10 +112,69 @@ def main() -> None:
                 f"| {count:,} |",
                 file=sink,
             )
+    # adding KPI code
+    # Currently hard-coded; what's a good alternative?
```

Comment:
There's already a constant that points to this: `LANGUAGE_SUMMARY_PATH`. (In fact this hard coding is wrong because it was changed a while back.)